nvidia-merged/PKGBUILD

```diff
@@ -105,6 +105,7 @@ DEST_MODULE_LOCATION[5]="/kernel/drivers/video"' \
 
     pushd "${_vgpupkg}"
     patch -p1 < "${srcdir}/nvidia-510.73.05-vgpu-5.18.patch"
+    patch -p1 < "${srcdir}/vgpu-vfio-flags.patch"
```

Comment:
This patch set NV_KVM_MIGRATION_UAPI to one for Kernel support of live migration. I thought it could be a universal patch for whatever flag we intend to change.

Reply:
Patches should be separated by feature, such as how 5.18 support is its own patch. Migration support should be specifically named that so if a user needs/wants to disable it, they simply have to comment the single line in the PKGBUILD instead of going and removing possibly multiple chunks out of a monolithic patch.